distributed/worker.py

```diff
@@ -1355,7 +1355,7 @@ async def delete_data(self, comm=None, keys=None, report=True):
             if report:
                 logger.debug("Reporting loss of keys to scheduler")
                 # TODO: this route seems to not exist?
-                await self.scheduler.remove_keys(
+                self.scheduler.remove_keys(
                     address=self.contact_address, keys=list(keys)
                 )
```

Comment:
If we don't await a coroutine then it won't occur.  So this code probably doesn't make sense.  

However, the TODO note just above it is interesting.  Maybe this code isn't necessary?  It might be worth looking at git blame here to see why it was added.

Reply:
It's even harder to await a function that doesn't exist, which is the case here. 😄 

With `git log -c -S remove_keys distributed/scheduler.py`, I managed to find that this function was removed somewhere in between https://github.com/dask/distributed/commit/dc5474815788f75b88be38a4d4242e515c48ef2b and https://github.com/dask/distributed/commit/2c175e86e0b1d43705510cf21ede408ccd71110d . Unfortunately that dates back to 2016, I believe this was a leftover from some commit around that time, and I can remove that line if it makes sense, but unfortunately I can't dig in deeper.

Reply:
Yeah I'd be +1 to just dropping this and the `TODO`.

Reply:
I tentatively dropped it on the newest commit, if there are any concerns with that I'll revert it.